views/js/qtiCreator/editor/gridEditor/droppable.js

```diff
@@ -174,28 +190,29 @@ define([
                 });
             }
 
-            var h = _resetColsHeight($col);
+            h = _resetColsHeight($col);
             $placeholder.css('height', '100%').parent().height(h);//causes issue on new-col sizes!
         };
 
         //recalculate the cols height according to new layout
-        var _resetColsHeight = function($col, self){
+        _resetColsHeight = function($col, asSelf){
 
             var maxHeight = 0;
-            if(self === undefined){
-                self = true;
+            var $cols;
+            if(!asSelf){
```

Comment:
I don't understand the usage of `asSelf` in this function.
I think that is possible to improve that.
What do you think?

Reply:
good catch!